README.md

```diff
@@ -2,3 +2,18 @@
 ![rust workflow](https://github.com/helium/proto/actions/workflows/rust.yml/badge.svg)
 ![node.js workflow](https://github.com/helium/proto/actions/workflows/node.yml/badge.svg)
 ![erlang workflow](https://github.com/helium/proto/actions/workflows/erlang.yml/badge.svg)
+
+## Contributing
+
+- Avoid `float` in Protobufs because representations differ across hardware architectures
+  + There are many floating point representations from IEEE, plus others
+  + i.e., some range of interior digits are random per float spec
+- Frequency should use `uint32`
+  + Locally, define accessor functions to return float after appropriate division
+- rssi or signal is always negative, thus use `sint32`
+- snr is signal-to-noise ratio and should be `uint32`
+- Fetch and share time in nanos, then truncate to appropriate granularity as needed
+  + e.g., get from OS in nanos
+- Reject any PR unless units are documented inline within Protobuf definition
+- Append units suffix on field names when deviating from the above descriptions
```

Comment:
no.. I'm still against this since we'd have to so many of them to have it not be some random handful.. 

```
Document units of fields
```

Reply:
Changed in commit a2fe4d5